server/engine.go

```diff
@@ -195,12 +195,71 @@ func (e *Engine) Close() {
 	e.clearInMemoryCaches()
 }
 
+// openIndexDB opens a connection to the index db, after performing the migrations
+// that and customizing the DB Connection limits
+func (e *Engine) openIndexDB() error {
+	_rootConfig := e.GetConfig()
+	genericConf, err := serviceconfig.GetDBAlias(_rootConfig.getGenericDbAlias())
+	if err != nil {
+		return fmt.Errorf("Could not find generic database: %v", err)
+	}
```

Comment:
You might have added this by mistake. As we should also not be managing the generic db's migrations, that should be handled by someone else.

Reply:
I agree with you there. However from what I could gather, search service was doing something with migrations and the generic database before I did anything here.

From my perspective, this sort of has me believing that this should be removed in a dedicated issue, rather than tacked onto this one

Reply:
interesting... ok, then I guess it has to be the way that it is now. Can't help but wonder who manages the search db's migrations. But yea, handle this in a different issue.